site/content/kapp/docs/develop/apply.md

```diff
@@ -67,6 +67,18 @@ Possible values: "" (default).
 
 Annotation value will be replaced with a unique ID on each deploy. This allows to force resource update as value changes every time.
 
+### kapp.k14s.io/renew-duration
+
+Available in v0.54.0+
+
+`kapp.k14s.io/renew-duration` annotation allows specifying an interval at which resource will get renewed. 
+
+Possible values: [ParseDuration](https://pkg.go.dev/time#ParseDuration).
+
+`kapp` injects an annotation with the current timestamp (`kapp.k14s.io/last-renewed-time=<current-time>`) to trigger an update if the specified duration has elapsed since the last renewal or since the resource has been created.
```

Comment:
This statement implies, that there is a server side which is triggering an update when the time has elapsed, which is not true.
Please reword to ensure we mention this will happen when kapp deploy is run - The update happens only when kapp deploy is called.